toml/src/test/kotlin/org/rust/toml/completion/LocalCargoTomlCompletionTestBase.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.toml.completion
+
+import org.intellij.lang.annotations.Language
+import org.rust.ide.experiments.RsExperiments
+import org.rust.openapiext.runWithEnabledFeature
+import org.rust.toml.crates.local.CargoRegistryCrate
+import org.rust.toml.crates.local.CargoRegistryCrateVersion
+import org.rust.toml.crates.local.withMockedCrates
+
+abstract class LocalCargoTomlCompletionTestBase : CargoTomlCompletionTestBase() {
+    protected fun crate(name: String, vararg versions: String): Crate =
```

Comment:
What do you think about using a static `@TestOnly` method instead of this function here and in `CrateNotFoundInspectionTest`? For example, something like:
```Kotlin
data class CargoRegistryCrate(val versions: List<CargoRegistryCrateVersion>) {
    companion object {
        @TestOnly
        fun of(vararg versions: String): CargoRegistryCrate =
            CargoRegistryCrate(versions.map {
                CargoRegistryCrateVersion(it, false, listOf())
            })
    }
}
```
and use `vararg crates: Pair<String, CargoRegistryCrate>` instead of `vararg crates: Crate`